### ISSUE TITLE
Add a 'unityVersion' parameter to the Request Activation File action

### ISSUE THREAD

**Issue by Jibbajabbafic** [webbertakken/unity-actions#30]
This would allow you to easily generate licenses for different versions.

It could be setup similar to the Build action like this:
```
      # Request manual activation file
      - name: Request manual activation file
        uses: webbertakken/unity-request-manual-activation-file@vX
        id: getManualLicenseFile
        with:
         unityVersion: 2019.X.XXXX
```

**Comment by webbertakken** [comment]
Reference https://github.com/webbertakken/unity-actions/issues/26#issuecomment-575554585

**Comment by webbertakken** [comment]
I am unsure if this is necessary however, as the activation file successfully works for multiple versions. 

Not sure if this is correct, but I think the activation file is requested per machine, not unity version. It's the activation with that file (which generates an active license) that's bound to a unity version.

**Comment by webbertakken** [comment]
I'm for now assuming that this is not necessary. If it is; it becomes part of this issue: https://github.com/webbertakken/unity-actions/issues/33

Closing this issue for now.

**Comment by webbertakken** [comment]
Turns out this actually is required!

See for more details: https://github.com/webbertakken/unity-actions/issues/37

**Comment by webbertakken** [comment]
Support for this has been released in [Request Activation File v1.1](https://github.com/webbertakken/unity-request-manual-activation-file/releases/tag/v1.1)
